src/sentry/testutils/cases.py

```diff
@@ -498,7 +498,7 @@ def __helper_enable_organization_2fa(self, organization, user, status_code, err_
         response = self.api_enable_org_2fa(organization, user)
         assert response.status_code == status_code
         if err_msg:
-            assert err_msg in response.content
+            assert err_msg.encode("utf-8") in response.content
```

Comment:
Is this correct? Or are we masking an error in that `response.content` has changed?

Reply:
django `HttpResponse.content` is always a bytestring. https://docs.djangoproject.com/en/1.11/ref/request-response/#django.http.HttpResponse.content